src/main_functions_batch.c

```diff
@@ -10,56 +10,73 @@
 #include "parser.h"
 #include "dictionary.h"
 #include "word.h"
-#include "main_functions_edit.h"
 #include "main_functions_batch.h"
+#include "main_functions_edit.h"
 #include "main_functions_home.h"
+#include "log.c/src/log.h"
 
 /* See main_functions_batch.h */
-char* edit_batch(char* line, dict_t* dict, int verbosity, int lnum)
-{
+char *edit_batch(char *line, dict_t *dict, int verbosity, int lnum) {
+    log_info("Batch mode started successfully.");
+
 	char *line_copy = strdup(line);
-    int max_no_suggestions = 2; //should the user decide this?
-    int length = strlen(line);
-    char **misspelled; //generates an empty array where the misspelled words in a line will be stored
+    int max_no_suggestions = 2; //Should the user decide this?
```

Comment:
we need to implement functionality that allows the user to decide this themselves, or at least set the constant to be higher (2 is too small)

Reply:
This is done in integration/suggestions. This is consistent to your own hard coded word.c in dev, so raising is not possible.